app/assets/stylesheets/flash_messages.scss

```diff
@@ -0,0 +1,8 @@
+.flash {
+  position: absolute;
+  top: 6%;
```

Comment:
6%? Would make more sense to use a pixel value, since you want to clear the fixed height navbar